adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBBannerRenderer.m

```diff
@@ -56,43 +53,45 @@ - (void)renderBannerForAdConfiguration:(nonnull GADMediationBannerAdConfiguratio
     NSError *error = GADMAdapterPangleErrorWithCodeAndDescription(
         GADPangleErrorInvalidServerParameters,
         [NSString stringWithFormat:@"%@ cannot be nil.", GADMAdapterPanglePlacementID]);
-    _loadCompletionHandler(nil, error);
+    self.loadCompletionHandler(nil, error);
     return;
   }
 
   NSError *error = nil;
   PAGBannerAdSize bannerSize = [self bannerSizeFormGADAdSize:adConfiguration.adSize error:&error];
   if (error) {
-    _loadCompletionHandler(nil, error);
+    self.loadCompletionHandler(nil, error);
     return;
   }
 
   PAGBannerRequest *request = [PAGBannerRequest requestWithBannerSize:bannerSize];
   request.adString = adConfiguration.bidResponse;
-
-  GADPangleRTBBannerRenderer *__weak weakSelf = self;
+  __weak typeof(self) weakSelf = self;
```

Comment:
Can you update this to `GADPangleRTBBannerRenderer *__weak weakSelf = self;`? The same applies to other ad formats.